### DISCUSSION TITLE
[PL] Jak uzyskać najbliższą wersję do dema gitstable?

### DISCUSSION THREAD

**Discussion by hahyes** [YetiForceCompany/YetiForceCRM#14412] [How to] [answered]
Witam,

mam ostatnio wiele problemów z 6.1 Service Pack v7 (np. zła edycja języków we wszystkich modułach go dotyczących, problemy z warunkami wyzwalaczy itd.). Widzę natomiast, że wersja gitstable działa bez zarzutu. Chciałbym pobrać zbliżoną lub taką samą wersję CRM, by sobie zhostować ją na własnej maszynie. Niestety nie wiem, z jakiego konkretnie brancha czy do którego commita jest wersja gitstable. Czy jest możliwe, aby sklonować z githuba taki odpowiednik gitstable?

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
Na gitstable jest to samo co na gałęzi https://github.com/YetiForceCompany/YetiForceCRM/tree/6.1.0 lub w tu https://github.com/YetiForceCompany/YetiForceCRM/releases/tag/6.1.0 
plus zainstalowany 6.1.0_ServicePack_v7.zip

**Reply by hahyes** [reply]
Dotychczas testowałem wszystko na hostingu zewnętrznej firmy i tam faktycznie na Service Pack v7 dzieją się najróżniejsze problemy. Instalowane przez Softaculous. Niestety nie jestem w stanie tam skonfigurować serwera w pełni tak, jak sugerowana jest konfiguracja w Yeti, ponieważ usługodawca nie udostępnia pełnej konfiguracji. Rozumiem, że z tego mogą być problemy z poprawnym działaniem. 

W każdym razie dziękuję za odpowiedź, jeżeli chodzi o gitstable.

**Reply by mariuszkrzaczkowski** [reply]
Bez prawidłowej konfiguracji serwera na pewno będą występować błędy, proszę zobaczyć https://yetiforce.com/pl/marketplace/hosting

**Comment by mariuszkrzaczkowski** [comment]
czy masz prawidłową konfigurację serwera ?
